### PR TITLE
Problem: missing amino json of Delegate, WithdrawDelegateReward (fix …

### PR DESCRIPTION
**Pull request by leejw51crypto**
Solution: add raw amino json

### CODE REVIEW

File: lib/src/transaction/msg/distribution/MsgWithdrawDelegatorReward.ts [hunk 1]
```diff
@@ -32,7 +32,13 @@ export const msgWithdrawDelegateReward = function (config: InitConfigurations) {
 
         // eslint-disable-next-line class-methods-use-this
         toRawAminoMsg(): legacyAmino.Msg {
-            throw new Error('Method not implemented.');
+            return {
+                type: 'cosmos-sdk/MsgWithdrawDelegatorReward',
```
Comment (eddycpc) [inline lib/src/transaction/msg/distribution/MsgWithdrawDelegatorReward.ts:36]: Can you add test case for these additions ?

**Review by eddycpc — CHANGES_REQUESTED** [review]

File: lib/src/transaction/msg/distribution/MsgWithdrawDelegatorReward.ts (hunk 1 above)
Comment (leejw51crypto) [reply lib/src/transaction/msg/distribution/MsgWithdrawDelegatorReward.ts:36]: ok,i will add unit test

**Comment by leejw51crypto** [comment]
added unit test for coverage

**Review by eddycpc — APPROVED** [review]

**Review by crypto-matto — APPROVED** [review]
LGTM

**Review by calvinaco — APPROVED** [review]
LGTM

File: lib/src/transaction/msg/distribution/MsgWithdrawDelegatorReward.ts (hunk 1 above)
Comment (calvinaco) [reply lib/src/transaction/msg/distribution/MsgWithdrawDelegatorReward.ts:36]: I assume this is resolved @eddycpc

**Pull request merged**